src/main/cpp/chassis/HolonomicDrive.cpp

```diff
@@ -127,6 +129,14 @@ void HolonomicDrive::Run()
             {
                 HoldPosition();
             }
+            else if (isRobotOriented)
```

Comment:
This doesn't work because soon as the button is not pressed you revert back to Field drive. We can fix it in the morning. I would say lets approve it

Reply:
do you mean pressing right stick won't work? you have to hold it down?